src/lib/apis/exchange.ts

```diff
@@ -0,0 +1,16 @@
+import urljoin from "url-join"
+import { assign } from "lodash"
+import fetch from "./fetch"
+import config from "config"
+
+const { EXCHANGE_API_BASE } = config
+
+export default (path, accessToken, fetchOptions = {}) => {
+  const headers = {}
+  if (accessToken) assign(headers, { Authorization: `Bearer ${accessToken}` })
```

Comment:
This is #super-minor, but can we[ expand this to multiline](https://github.com/artsy/metaphysics/blob/4a5c082f5c94a6f62d88b56286c6cb434c4da716/src/lib/apis/convection.ts#L11-L13)? 

Lots of subtle bugs potentialy happen when functionality is packed into one-line ifs, and it hurts debugging. (Fine if we're just `returning`)